platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -792,19 +820,19 @@ - (id)mgl_jsonExpressionObject {
                 }
 
                 return expressionObject;
-            } else if ([function isEqualToString:@"mgl_match:"]) {
-                NSMutableArray *expressionObject = [NSMutableArray arrayWithObjects:@"match", self.operand.mgl_jsonExpressionObject, nil];
+            } else if ([function isEqualToString:@"MGL_MATCH"]) {
```

Comment:
This removes support for the conventional custom function `mgl_match:`. In this PR, I’ve left in support for those functions, just in case aftermarket expressions break with a new iOS SDK version. (I’ve equivocated about whether to remove public documentation about the conventional functions, but the implementations remain.) See the methods at the bottom of this file for an example of how to support both versions of the method simultaneously.